wit/mat.wit

```diff
@@ -21,6 +21,9 @@ interface mat {
         /// Create a new Mat with the specified size and type.
         new-mat-with-size: static func(cols: u32, rows: u32, mattype: mattype) -> mat;
 
+        /// Clone returns a cloned full copy of the Mat.
```

Comment:
I can't leave a comment above, but resource types can have constructors and an implicit resource-drop method, which could take the place of close().

Reply:
To match the semantics of how the OpenCV `Mat` type is handled that is how we currently handle it in the GoCV bindings with a separate `Close()` method.